ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,388 @@
+# ERC-X: Consolidated Dynamic Storage in Solidity  
```

Comment:
```suggestion
---
eip: 7844
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.